system_test.go

```diff
@@ -0,0 +1,18 @@
+package glutton
+
+import (
+	"testing"
+)
+
+func TestCountOpenFiles(t *testing.T) {
+	if !(countOpenFiles() > 0) {
```

Comment:
```suggestion
	if countOpenFiles() < 0 {
```

Reply:
I used the greater than 0 because you said it would be better if we test if the function works...... And therefore I kept my test positive :^^

Reply:
`<=` then